modules/ts/src/ts_perf.cpp

```diff
@@ -1691,6 +1943,10 @@ void TestBase::RunPerfTestBody()
 {
     try
     {
+#ifdef ENABLE_INSTRUMENTATION
+        cv::instr::setFlags(cv::instr::getFlags()|cv::instr::FLAGS_MAPPING); // enable mapping for first run
+        cv::instr::queueCommand(cv::instr::COMMAND_RESET); // delayed clean up. actual execution before timer start
```

Comment:
Also some tests do not use macros but they all use startTimer function, so I need to reset trace just before timer is set, but only once.